src/pages/gen1/[platform]/build-a-backend/functions/secrets/index.mdx

```diff
@@ -89,6 +93,25 @@ const { Parameters } = await (new aws.SSM())
 
 // Parameters will be of the form { Name: 'secretName', Value: 'secretValue', ... }[]
 ```
+</Block>
+
+<Block name='AWS SDK V3'>
+
+```js
+const {ssm} = require("@aws-sdk/client-ssm");
```

Comment:
can we pivot this example to use the SSMClient from this package and the GetParametersCommand? 

https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/client/ssm/
https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/client/ssm/command/GetParametersCommand/